assets/themes/zeppelin/css/style.css

```diff
@@ -88,32 +99,41 @@ body {
 .navbar {
   background: #3071a9;
   border-bottom: 0px;
-  height: 50px;
+  height: 60px;
+  -webkit-box-shadow: 0 8px 6px -6px #999;
```

Comment:
1. Let's unify box-shadow value. 
2. 0px 2px 2px rgba(0, 0, 0, 0.4) would be better. Currently it's too much as @AhyoungRyu mentioned.